src/game/spawn/mod.rs

```diff
@@ -0,0 +1,11 @@
+//! Handles spawning of entities. This is typically done through
```

Comment:
"typically" according to whom? Observers were only just added :P

Reply:
According to us. We are the authority here :P

Reply:
[`7945346` (#70)](https://github.com/TheBevyFlock/bevy-template/pull/70/commits/79453462dd6fabb65ccad4203f0e2d2938147e0c)

Reply:
I mean that this makes it sound like it's an old tradition in bevy.